pkgs/top-level/python-packages.nix

```
@@ -2733,6 +2733,10 @@ in {
 
   dask = callPackage ../development/python-modules/dask { };
 
+  dask-gateway = callPackage ../development/python-modules/dask-gateway { };
+
+  dask-gateway-server = callPackage ../development/python-modules/dask-gateway-server { };
```

future proof from a python3Packages.go package being added
```suggestion
  dask-gateway-server = callPackage ../development/python-modules/dask-gateway-server {
    inherit (pkgs) go;
  };
```

---

done